ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -51,14 +51,15 @@ struct S3FileSystem;
 // Statistics to be collected for the S3 storage.
 struct S3Statistics {
     // Operations using AWS SDK.
-    uint64_t listObjectsCount;  // Number of S3 list objects requests
-    uint64_t putObjectCount;    // Number of S3 put object requests
-    uint64_t getObjectCount;    // Number of S3 get object requests
-    uint64_t objectExistsCount; // Number of S3 object exists requests
+    std::atomic<int64_t> listObjectsCount;  // Number of S3 list objects requests
```

Comment:
Two questions here: (1) was it intentional for these to become signed? And (2) why do these now need to be atomic? (Or - did they always need to be atomic, and this is correcting them?)

Reply:
They are intentionally signed, as they will eventually map to WiredTiger stats, which are also signed. Also, yes, they needed to be atomic as they get incremented and accessed concurrently. We are correcting them now.

Reply:
Got it - thanks.